python_tsp/heuristics/local_search.py

```diff
@@ -78,12 +72,20 @@ def solve_tsp_local_search(
         improvement = False
         for n_index, xn in enumerate(neighborhood_gen[perturbation_scheme](x)):
             if default_timer() - tic > max_processing_time:
-                logger.warning("Stopping early due to time constraints")
+                warning_msg = "WARNING: Stopping early due to time constraints"
+                if log_file:
+                    print(warning_msg, file=log_file_handler)
+                print(warning_msg)
```

Comment:
If we opt to follow the suggestions in [this comment](https://github.com/fillipe-gsm/python-tsp/pull/20#issuecomment-1194160693), then the only last suggestion would be to only print the warning if `verbose` is set to `True`:

```suggestion
                if verbose:
                    print(warning_msg)
```

Even if the warning is important, it may be weird to see something in the stdout even by setting `verbose` as `False`.

So just this modification on both files would be enough.